elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -86,32 +83,58 @@ $variables: (
 
     border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} #{pfe-local($cssvar: BorderColor, $region: inner)};
     border-radius: 2px;
-    
     outline: none;
   }
 }
 
+
 ::slotted(*:not(:disabled)) {
   cursor: pointer;
 }
 
-.pfe-cta--arrow {
-  display: #{pfe-local($cssvar: arrow--Display, $region: arrow)};
-  padding: #{pfe-local($cssvar: Padding, $region: arrow)};
-  fill:    #{pfe-local(Color)};
-  position: relative;
-  top:    .1rem;
-  width:  13px;
-  height: 13px;
-  transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+$arrow-width: 30px;
```

Comment:
@LyndseyR this is a mess of a PR, ignore everything except for this little section here. This is the idea I was playing with, which worked in Chrome and IE11, but in Firefox the arrow was on a new line :( 

The concept is that I was using a pseudo element to create space after the link, then using a negative margin to pull the SVG into that space.

Reply:
Note to self, review this first https://github.com/patternfly/patternfly-elements/pull/630/files